packages/wallets/src/components/WalletCashierHeader/WalletCashierHeader.tsx

```diff
@@ -14,12 +14,11 @@ const WalletCashierHeader = () => {
     const { data } = useActiveWalletAccount();
     const { isMobile } = useDevice();
     const history = useHistory();
-    const { currency, currency_config, display_balance, landing_company_name, wallet_currency_type } = data || {};
     const location = useLocation();
 
     return (
         <WalletGradientBackground
-            currency={currency_config?.display_code || 'USD'}
+            currency={data?.currency_config?.display_code || 'USD'}
```

Comment:
const { currency_config , currency, landing_company_name, ...} = data

Reply:
Because of the camelcase i am not destructuring.